README.markdown

```diff
@@ -240,6 +241,14 @@ Sets the timeout (in ms) protection for subsequent operations, including the `co
 
 [Back to TOC](#table-of-contents)
 
+set_timeouts
+----------
+`syntax: red:set_timeouts(connect_timeout, send_timeout, read_timeout)`
+
+Sets each timeouts (in ms) protection for subsequent operations more exactly.
```

Comment:
Grammar: "each timeouts protection" => "each timeout threshold". "more exactly"?

Should be "with finer granularity than `set_timeout()`" instead?

Reply:
granularity is that word i mean